_container_scripts/keyper-db-init.sh

```diff
@@ -2,4 +2,15 @@
 
 set -e
 
-createdb -U postgres keyper
+echo "Checking for backup dump file..."
+if [ -f "/var/lib/postgresql/dump/keyper.dump" ]; then
+    echo "Backup dump found, restoring database with full schema and data..."
+    # Create the database first
+    createdb -U postgres keyper 2>/dev/null || echo "Database already exists"
+    # Restore the database with full schema and data
+    pg_restore -U postgres -d keyper --create --clean -v /var/lib/postgresql/dump/keyper.dump
```

Comment:
The database name given with `-d` is used to issue the create statements from the `--create` option. Therefore it should be `postgres` here.